gh-url.el

```diff
@@ -152,8 +152,10 @@
            (gh-url-run-request req resp)))))))
 
 (defun gh-url-form-encode (form)
-  (mapconcat (lambda (x) (format "%s=%s" (car x) (cdr x)))
-             form "&"))
+  (if (stringp form)
+      form
+    (mapconcat (lambda (x) (format "%s=%s" (car x) (cdr x)))
+               form "&")))
```

Comment:
This looks hacky at first glance but it seemed like a better idea than going through all the effort to change the string to a list of dot pairs just to deconstruct it later on. Any other ideas?